osgar/platforms/yuhesen.py

```diff
@@ -192,6 +199,17 @@ def on_can(self, data):
             self.pose_counter = 0
             self.publish_pose2d(self.last_left_speed, self.last_right_speed)
 
+    def on_desired_steering(self, data):
+        speed_mm_per_sec, steering_deg_msec = data
```

Comment:
The unit is not milliseconds.

Reply:
good point, thanks! How is 1/100th typically named? Percent? Hundredth? `steering_deg_1_100th`? Suggestions welcome (fallback is `steering` without units), thanks

Reply:
I don't know, meaning variables is not my strong point.  `steering_deg_hundredth` or just `steering_deg100`? Anyway, we should add describing of the `desired_steering` and `desired_ speed` to documentation including units.